### PR TITLE
Add: Added Cart Page

### PR DESCRIPTION
**Pull request by JaswanthReddyPadala**
Added Cart Page

image

### CODE REVIEW

File: styles/cart.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  
+  .q-cart-total-container {
+    background-color: var(--white);
+    padding: var(--spacing-1);
+    margin-left: var(--spacing-1);
+    width: 45%;
+  }
+  
+  .q-cart-details > * {
+    margin: 0.75rem 0;
```
Comment (sagargse) [inline styles/cart.css:18]: use px value if less than 1 rem

File: styles/cart.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  }
+  
+  .q-cart-total-details {
+    padding: 0.7rem 0;
```
Comment (sagargse) [inline styles/cart.css:22]: same thing as above

File: pages/cart.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (sagargse) [inline pages/cart.html:7]: just a suggestion you can use author attribute also under meta

**Review by sagargse — COMMENTED** [review]
else all looks good to me do the changes and you can go ahead

**Review by sagargse — APPROVED** [review]

File: styles/cart.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+      }
+      .q-cart-total-container {
+        width: 100%;
+        margin: 0;
+        margin-top: var(--spacing-2);
+        margin-bottom: var(--spacing-2);
+      }
+    
+      .q-cart-main {
+        margin: 2.5% 1rem;
```
Comment (gadgetfather) [inline styles/cart.css:81]: Try using same units

**Review by gadgetfather — APPROVED** [review]
- Good use of class names
- very good UI
-  Try using same units for css values

**Pull request merged**